aapcs64/aapcs64.rst

```diff
@@ -341,16 +341,16 @@ VG
    the number of bits in an SVE vector register divided by 64.
 
 ILP32
-   SysV-like data model where int, long int and pointer are 32-bit
+   SysV-like data model where int, long int and pointer are 32-bit.
 
 LP64
    SysV-like data model where int is 32-bit, but long int and pointer are 64-bit.
 
 LLP64
    Windows-like data model where int and long int are 32-bit, but long long int and pointer are 64-bit.
 
-This document uses the following terms.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This Document Uses the Following Terms
```

Comment:
This should not be a title, but an ordinary sentence.

This document uses the following abbreviations:

Reply:
Done. 1a02375